src/makers/nebra/hotspots.ts

```diff
@@ -25,6 +25,7 @@ const IN1BLE = {
   },
   antenna: {
     us: ANTENNAS.NEBRA_US_3,
+    eu: ANTENNAS.NEBRA_EU_3,
```

Comment:
don't we sell outside of eu and us?

Reply:
This is about default antenna. We can add China default antenna here. Also the users can add custom antenna anyway.

Reply:
no need. I was just curious.